fda-model-test/PairedDataTest.cs

```diff
@@ -1,29 +1,44 @@
 using NUnit.Framework;
 using paireddata;
+using System.Collections.Generic;
 
 namespace fda_model_test
 {
     public class PairedDataTest
     {
-        public static double[] xs = { 0, 2000, 5000, 10000, 25000, 50000 };
-        private static double[] ys = { 556, 562, 565, 566, 570, 575 };
-        PairedData myPairedData = new PairedData(xs, ys);
+        static List<double> Flows = new List<double> { 0, 2000, 5000, 10000, 25000, 50000 };
+        static List<double> Stages = new List<double> { 556, 562, 565, 566, 570, 575 };
+        PairedData myRatingCurve = new PairedData(Flows, Stages);
 
-        [SetUp]
-        public void Setup()
+        static List<double> Probabilities = new List<double> { .5, .2, .1, .05, .02, .01 };
+        PairedData myFlowFrequencyCurve = new PairedData(Probabilities, Flows);
+
+
+        [Test]
+        public void GetXvals()
         {
- 
+            Assert.AreEqual(Flows, myRatingCurve.Xvals);
         }
-
         [Test]
-        public void TestGetXvals()
-        { 
-            Assert.AreEqual(xs,myPairedData.Xvals);
+        public void GetYvals()
+        {
+            Assert.AreEqual(Stages, myRatingCurve.Yvals);
+        }
+        [Test]
+        public void FInterpolatesCorrect()
+        {
+            double expected = 565.5; //linear interp of ys when x=7500
```

Comment:
i would use a more clear case if your x values and y values are 1 to 1 we should get back equivalent values, if they are 1 to 2 we get back double our intput. those are really easy to test.

Reply:
But this is great this is an excellent test we need to verify. We need to test more cases, over, under, exact values, inbetween values.